pinc/faq.inc

```diff
@@ -97,7 +97,8 @@ $Stealth_Scannos_URL = make_forum_url("t", $Stealth_Scannos_id, 1563);
 $Proofing_Interface_Help_URL = make_forum_url("t", $Proofing_Interface_Help_id, 2739);
 $XHTML_suggestion_URL = make_forum_url("t", $XHTML_suggestion_id, 4405);
 
-function make_forum_url($type, $id, $default)
+/** @param string|int $id */
+function make_forum_url(string $type, $id, string $default): string
```

Comment:
After doing some inspection of the code, let's just make this:
```suggestion
function make_forum_url(string $type, int $id, string $default): string
```
That's accurate with how the function is used. I'll do a fast-follow PR that will do a lot of cleanup of the variables above which we no longer need because they are not used.